lib/Doctrine/MongoDB/Event/AggregateEventArgs.php

```diff
@@ -58,9 +64,26 @@ public function getPipeline()
 
     /**
      * @since 1.2
+     * @return array
      */
     public function getOptions()
     {
         return $this->options;
     }
+
+    /**
+     * @param array $pipeline
```

Comment:
Since we add new public methods in a backwards compatible way, Semantic Versioning requires this change to be in a [minor version increment](http://semver.org/#spec-item-7). That makes it `@since 1.3`.